src/scripts/move-popup/dimMoveItemProperties.directive.js

```diff
@@ -129,12 +134,53 @@ function MoveItemPropertiesCtrl($sce, $q, dimStoreService, dimItemService, dimSe
     });
   };
 
+  vm.openDiscuss = function() {
+    ngDialog.closeAll();
+    $rootScope.$broadcast('dim-store-item-discuss', {
+      item: vm.item
+    });
+  };
+
   vm.updateNote = function() {
     if (angular.isDefined(vm.item.dimInfo.notes)) {
       vm.item.dimInfo.save();
     }
   };
 
+  vm.reviewBlur = function() {
+    var item = vm.item;
+    var userReview = vm.toUserReview(item);
+
+    dimDestinyTrackerService.updateUserRankings(item,
+                                                userReview);
+  };
+
+  vm.toUserReview = function(item) {
+    var newRating = item.userRating;
+    var review = item.userReview;
+    var pros = item.userReviewPros;
+    var cons = item.userReviewCons;
+
+    var userReview = {
+      rating: newRating,
+      review: review,
+      pros: pros,
+      cons: cons
+    };
+
+    return userReview;
```

Comment:
Yeah, I copypasta'd a bunch of things between move item and the item discussion pane.

This could be moot after the UI change?